src/main/scala/devices/spi/SPI.scala

```diff
@@ -93,14 +93,14 @@ object SPI {
     InModuleBody { qspiNode.makeIO()(ValName(qspi.name)) }
   }
 
-  def synchronize(q: SPIPortIO): SPIPortIO = {
-    val x = Wire(new SPIPortIO(q.c))//, DontCare)
+  def connectPort(q: SPIPortIO): SPIPortIO = {
+    val x = Wire(new SPIPortIO(q.c))
     x.sck := q.sck
```

Comment:
I guess this could again just be x <> q, since we are no longer adding the flops

Reply:
I wanted to touch minimal logic  - but I guess no option :)

Reply:
Updated now.